helper/date_format.go

```diff
@@ -50,6 +50,16 @@ func DateFormatYYYYMMDDNoSlash(s string) string {
 	return template.HTMLEscapeString(t.Format("20060102"))
 }
 
+func DateTimeOnsDatePatternFormat(s string) string {
```

Comment:
Naming stuff is hard. This will probably have to change further down the line as we adopt the design systems's patterns more and more and our existing helpers become legacy. However, I appreciate that is feature creep and quite a large task to rename helper methods over the estate